crashlogging/src/main/java/com/automattic/android/tracks/crashlogging/internal/SentryCrashLogging.kt

```diff
@@ -75,6 +76,10 @@ internal class SentryCrashLogging constructor(
                     appendExtra(event)
                     event
                 }
+                sampleRate = when (val errorsSampleRate = dataProvider.errorSampling) {
+                    ErrorSampling.Disabled -> null
```

Comment:
The `null` here confused me a bit, but looking at the code it's clear that this is a sort of mapping between the exposed `ErrorSampling` and this internal `option` from Sentry, I guess?
No change required, I just want to confirm my understanding 🙂 (but maybe worth a comment if it's not that obvious)

Reply:
> The null here confused me a bit, but looking at the code it's clear that this is a sort of mapping between the exposed ErrorSampling and this internal options from Sentry, I guess?

Yes, exactly! Looking at source code of Sentry, `options.sampleRate` is `null` by default and `null` means "we don't sample errors". 

Alternatively, `null` is the same as we would set `sampleRate` = `1.0` (meaning: all errors are sent), but `null` is a default expected by framework to disable this feature, so that's why I decided to map like this.